### DISCUSSION TITLE
Adding the option "other" to recoder device type

### DISCUSSION THREAD

**Discussion by Valentina-Fed** [LAAC-LSCP/ChildProject#349] [Q&A]
Hi, 

I would like to ask you, if it is possible to add the option "other" to the recorder device type. I am working now the a dataset where the type of recoder is not mentioned. 

Thank you!

**Comment by lucasgautheron** [comment]
Hello,

sure! I've added "unknown" instead (for cases where it is known but not any of the most common values, i'd say it is better to specify the known value rather than "other")

https://github.com/LAAC-LSCP/ChildProject/commit/898a7e99bb66f3add39b790f38466c85dc8dab98
